R/do_adjustRtime-functions.R

```diff
@@ -582,3 +582,418 @@ adjustRtimeSubset <- function(rtraw, rtadj, subset,
     }
     rtadj
 }
+
+###########################################################
+###### LamaParama
+
+#' @title Landmark-based alignment: aligning a dataset against an external
+#' reference
+#'
+#' @aliases LamaParama-class
+#'
+#' @description
+#' Alignment is achieved using the ['adjustRtime()'] method with a `param` of
+#' class `LamaParama`. This method #' corrects retention time by aligning
+#' chromatographic data with an external reference dataset. Peaks in the
+#' experimental chromatographic data are aligned to predefined landmark
```

Comment:
maybe separate here the matching from the alignment?

*... dataset. Chromatographic peaks in the experimental data are first matched to predefined (external) landmark features based on their mass-to-charge ratio and retention time and subsequently the data is aligned by minimizing the differences in retention times between the matched chromatographic peaks and lamas.*